packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -614,12 +614,17 @@ export function TableNG(props: TableNGProps) {
         rows={paginatedRows}
         headerRowClass={clsx(styles.headerRow, { [styles.displayNone]: noHeader })}
         headerRowHeight={headerHeight}
-        onCellClick={({ column, row }, { clientX, clientY, preventGridDefault, target }) => {
+        onCellClick={({ column, row }, { clientX, clientY, preventGridDefault, target, currentTarget }) => {
           // Note: could be column.field; JS says yes, but TS says no!
           const field = columns[column.idx].field;
-          const targ: HTMLElement = target as HTMLElement;
 
-          if (colsWithTooltip[getDisplayName(field)] && targ.matches('a') && !targ.hasAttribute('href')) {
+          // find the tooltip control anchor tag within the cell. if it doesn't exist, this cell can't trigger tooltips.
+          const tooltipControl = currentTarget.querySelector('a[aria-controls="table-data-links-actions-tooltip"]');
```

Comment:
one thing I noticed: I think on the current branch, the links don't work correctly if you click within a BarGaugeCell or some other more complex cell, because `target` isn't guaranteed to be our `a` tag, but could be something more deeply nested. I updated this logic to grab `currentTarget` (the cell), find the tooltip anchor (gave it an `aria-controls` and applied the `id` to the Portal'd div), and if it exists, check if the target is with the anchor.